README.md

```diff
@@ -127,7 +127,7 @@ Once you've started an `sbt` session you can run one of the core commands:
       to skip generating / publishing API docs (speeds up the process).
 
 If a command results in an error message like `a module is not authorized to depend on
-itself`, it may be that a global SBT plugin (such as [ENSIME](https://ensime.github.io/)) is
+itself`, it may be that a global SBT plugin is
 resulting in a cyclical dependency. Try disabling global SBT plugins (perhaps by
```

Comment:
```suggestion
resulting in a cyclical dependency. Try disabling global sbt plugins (perhaps by
```

Reply:
They should have kept it as an acronym for "SBT Build Tool".